source/channels/make-calls.rst

```diff
@@ -0,0 +1,77 @@
+
+Limitations
+-----------
+
+- In Mattermost Cloud, up to eight participants in each channel can join a call.
+- In Mattermost self-hosted, the recommended maximum number of participants per server is 200. This can be changed in the ``config.json`` file.
+
+Configuration
+-------------
+
+For Mattermost self-hosted customers, the calls plugin needs to be enabled in the System Console. Additional configuration is also available via the System Console.
+
+Frequently asked questions
+--------------------------
+
+Can I password-protect a call?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No. Any member with sufficient permission to access the channel will be able to join the call.
+
+Is there encryption?
+~~~~~~~~~~~~~~~~~~~~
+
+Media (audio/video) is encrypted using security standards as part of WebRTC. It's mainly a combination of DTLS and SRTP. It's not e2e encrypted in the sense that in the current design all media needs to go through Mattermost which acts as a media router and has complete access to it. Media is then encrypted back to the clients so it's secured during transit. In short: only the participant clients and the Mattermost server have access to unencrypted call data.
+
+What are the potential performance impacts?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Database load should be minimal. Overall instance load however will be affected, especially CPU usage, growing as a function of the number of participants produce the number of active tracks (unmuted participants and users sharing their screen). Screen sharing has the highest impact on both CPU and bandwidth. The latter can be more easily estimated as the audio/video bitrates are constrained and predictable (around 40-60Kbps for each audio track and up to 1Mbps per screen track).
+
+Are there any third-party services involved?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+STUN servers are configurable on the plugin itself. The default one used is:
+
+- stun:stun.global.calls.mattermost.com:3478
+
+Depending on the setup they may not be necessary (e.g. if running a single instance and providing a ICE Host Override). 
+
+No media goes through STUN servers, the only sensitive information that passes through is the client's (and server's) public IP address.
+
+Troubleshooting
+---------------
+
+My call is disconnected after a few seconds and I can't transmit voice nor hear anything.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is usually a sign that the underlying UDP channel has not been established and the connection timeouts after ~10 seconds. When the connection has been established correctly an `rtc: connected` line should appear in the client-side logs (JS Console).
```

Comment:
@streamer45, I've left this in but not sure if we need this section/question? It doesn't provide remediation so it seems unhelpful.